packages/validator/src/services/indices.ts

```diff
@@ -14,6 +14,26 @@ const PUBKEYS_PER_REQUEST = 10;
 // To assist with readability
 type PubkeyHex = string;
 
+// To assist with logging statuses, we only log the statuses that are not active_exiting or withdrawal_possible
+type SimpleValidatorStatus = "pending" | "active" | "exited" | "withdrawn" | "unknown";
+
+const statusToSimpleStatusMapping = (status: routes.beacon.ValidatorStatus): SimpleValidatorStatus => {
+  if (status.includes("active") || status === "withdrawal_possible") {
+    return "active";
+  }
+  if (status.includes("exiting")) {
+    return "exited";
+  }
+  if (status.includes("pending")) {
+    return "pending";
+  }
+  if (status === "withdrawal_done") {
+    return "withdrawn";
+  }
+
+  return "unknown";
```

Comment:
Every ValidatorStatus has a SimpleValidatorStatus that it maps to, there shouldn't be an unknown